examples/sandbox/index.ts

```diff
@@ -10,6 +10,7 @@ import {
 } from '@shapeshiftoss/hdwallet-core'
 
 import { isKeepKey } from '@shapeshiftoss/hdwallet-keepkey'
+import { isPortis } from '@shapeshiftoss/hdwallet-portis'
```

Comment:
umm how come this is here ?

Reply:
The portis devs made a breaking change to line 352 (`coin: "Bitcoin"`) when the coin is really Ethereum because they need to pretend it is Bitcoin for the way their wallet works. So I had to add `coin: isPortis(wallet) ? "Bitcoin" : "Ethereum"`

Reply:
gotcha 👌 / thank you!

Reply:
The real bug here is that we're asking for an xpub with `Ethereum` script type. There is no such thing as an Ethereum xpub. Only a Bitcoin-network-magic xpub with Ethereum's bip44 path.